.github/workflows/ci-proto.yml

```diff
@@ -22,6 +22,16 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+      - name: Set up Python
```

Comment:
Is it possible to use Buf instead of these new dependencies?

Reply:
seconded, though can be done in a later pr.

Reply:
ENG-1559
will handle in the future
it's not that simple and requires code changes 

naive try changes the code generation too much (introduce connect, break linting and more)